src/classes/Globals.js

```diff
@@ -155,7 +155,7 @@ class Globals {
 		const ua = parser(navigator.userAgent);
```

Comment:
@philipp-classen how about we bump parser version?

Reply:
> @philipp-classen how about we bump parser version?

You mean the ua-parser-js (https://github.com/faisalman/ua-parser-js), right? 1.0.2 is still the most recent version, so if user agents such as the ones in ChromeOS are not detected,  perhaps creating an issue there is an idea.

Reply:
Creating an issue there is a good idea, but is not feasible in the short term sense

Reply:
Yes, true. Let's merge your quick fix in the meantime.

Reply:
@philipp-classen I think I fixed it 😄